core/templates/pages/profile-page/profile-page.component.css

```diff
@@ -101,20 +101,6 @@
   width: 90%;
 }
 
-.oppia-profile-username-large-screen {
-  display: block;
-  font-size: 130%;
-  margin-top: 20px;
-  text-align: center;
-}
-
-.oppia-profile-username-small-screen {
-  display: none;
-  font-size: 110%;
-  margin-top: 20px;
-  max-width: 30px;
-  text-align: left;
-}
```

Comment:
Why are we removing these classes? After removal how are we handling this because I couldnt find any media query for font-change and allignment. Also can you please provide a proof showing the behaviour of the page at different screen size?

Thanks